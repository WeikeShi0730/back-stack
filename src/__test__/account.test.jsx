import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Account from "../pages/account/account.component";

describe("<Header />", () => {
  test("<Header />", () => {
    const { baseElement } = render(<Account />);
    expect(baseElement).toMatchSnapshot();
  });
});
