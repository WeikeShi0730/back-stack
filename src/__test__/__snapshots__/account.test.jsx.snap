// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> <Header /> 1`] = `
<body>
  <div>
    <div
      class="h-full flex flex-col justify-center items-center"
    >
      <div
        class=""
      >
        <div
          class="mt-10"
        >
          <button
            class="text-xs md:text-sm lg:text-base py-2 px-4 text-red-500 rounded border border-red-500 font-light"
          >
            Sign out
          </button>
        </div>
      </div>
    </div>
  </div>
</body>
`;
